JuiceMaker/JuiceMaker/Controller/JuiceOrderViewController.swift

```diff
@@ -44,6 +58,11 @@ class JuiceOrderViewController: UIViewController {
         }
     }
     
+    @IBAction func touchUpModifyButton(_ sender: UIBarButtonItem) {
+        self.showModifyingInventoryView()
```

Comment:
```suggestion
        self.showModifyingInventoryViewController()
```

다른 팀원이 코드만 보고 파악할때 진짜 `View`라고 느껴질 수 있을 것 같아요 ㅎㅎ 간결함보다는 확실함이 개발에서는 더 좋다고 생각합니다

Reply:
수정하였습니다!